README.md

```diff
@@ -1,10 +1,71 @@
 # `incuna-groups` [![Build Status](https://magnum.travis-ci.com/incuna/incuna-groups.svg?token=9QKsFUYHUxekS7Q4cLHs&branch=master)](https://travis-ci.org/incuna/incuna-groups)
 
-An extensible Django app, that provides forum functionality.
-- Create groups.
-- Create discussions on groups.
-- Comment on discussions.
-
-Email reply gotchas:
-- The `/groups/reply/` endpoint _has a trailing slash_.  Ensure that this slash is included in any Mailgun config otherwise you'll get a stream of HTTP301s.
-- If you're using `incuna_auth.LoginRequiredMiddleware`, make sure to add `/groups/reply/` to `LOGIN_EXEMPT_URLS` to avoid more 301s.
+An extensible Django app that provides forum functionality.
+- Administrators can create discussion groups.
+- Users can create discussions on groups, and comment on those discussions.
+- Users can also subscribe to groups and/or discussions to receive notifications from them, and post responses by replying to the notification emails. 
+
+## Installation
+
+`incuna-groups` is on PyPI, so you can install it with `pip install incuna-groups`.
+
+This project contains migrations, so run `python manage.py migrate` before using it.
```

Comment:
Hahah, what do you know - having found that API description, I got a file upload on the first try.  Shazam.